### PR TITLE
Cleans up unused or shitty mob defines

### PR DESCRIPTION
**Pull request by DominikPanic**

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -883,17 +881,6 @@
 
 	return species.name
 
-/mob/living/carbon/human/proc/play_xylophone()
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/human/human.dm]: no, this is a blocker, this proc must remain.

Comment (psykzz) [reply code/modules/mob/living/carbon/human/human.dm]: https://youtu.be/XVQOhIVcMj0

Comment (Rohesie) [reply code/modules/mob/living/carbon/human/human.dm]: This is a good feature. It should stay, even though very spooky.

Comment (DominikPanic) [reply code/modules/mob/living/carbon/human/human.dm]: Poe's law

Comment (Rohesie) [reply code/modules/mob/living/carbon/human/human.dm]: Talking more seriously, keep it there. It's funny enough.

**Comment by Rohesie** [comment]
Merge conflict. Other than that and the change request, the PR is fine.

File: code/modules/mob/living/carbon/human/human.dm (hunk 1 above)
Comment (DominikPanic) [reply code/modules/mob/living/carbon/human/human.dm]: It's not even usable and relies on non-existant dna code

Comment (Rohesie) [reply code/modules/mob/living/carbon/human/human.dm]: It can be invoked by admins. /tg/ has skeletons.
This is a little gem, nothing too evil.

Comment (DominikPanic) [reply code/modules/mob/living/carbon/human/human.dm]: Nothing uses it and you know our stance on that. So far it has been used exactly 0 times, procs that aren't used anywhere but are just for admins are very unlikely to be ever used.

**Pull request merged**